integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -262,7 +351,7 @@ def export(self, **kwargs):
 
             # We only need to fetch content metadata if there are items to update or create
             if items_to_create or items_to_update:
-                items_create_keys = [key.get('content_key') for key in items_to_create]
+                items_create_keys = list(items_to_create.keys())
```

Comment:
we're now passing a dict of content_key : `ContentMetadataItemTransmission` item